package.json

```diff
@@ -93,7 +93,7 @@
     "syncpack": "npx syncpack fix-mismatches && npx syncpack format && npx syncpack set-semver-ranges",
     "test": "jest --maxWorkers=50% --config=frontend/jest.config.js",
     "test:ci": "jest --ci --coverage --max-workers=2 --cacheDirectory=$(yarn config get cacheFolder) --config=frontend/jest.config.js",
-    "typecheck": "yarn workspaces foreach -A run typecheck",
+    "typecheck": "tsc --noEmit -p frontend",
```

Comment:
Can we face challenges by changing this to use -p frontend? This means that it reads from the tsconfig in the root of the frontend folder, right? If so, any custom settings in the different workspaces will be excluded, won't they? 🤔

What will happen if we create a component in @studio/components? Will the system detect a type error in the component even if it is not in use in `app-development`, but just created and available in studio-components?

Reply:
This one is a quick temporary fix and it's currently working because none of our `tsconfig.json` files have custom configurations, they just point to the tsconfig of the frontend folder. I wanted to take a closer look and fix it per workspace in a separate issue: https://github.com/Altinn/altinn-studio/issues/13031

Reply:
But I agree, this solution is not ideal. I can revert it and address the fix in the separate issue.